invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMemberBulkActions.js

```diff
@@ -25,7 +23,10 @@ class ManagerMemberBulkActionsCmp extends Component {
     this.state = {
       modalOpen: false,
       currentAction: undefined,
+      // eslint-disable-next-line react/no-unused-state
       role: undefined,
+      // eslint-disable-next-line react/no-unused-state
+      visible: undefined,
       selectedMembers: {},
```

Comment:
For reviewers: Destructuring the properties in a dynamic way (line 119) bugs the eslint. Making it think that the props are not being used, so and eslint-disable es needed. The alternative would be to not use destructuring in that case, but then we would ignore the destructuring warnings instead. In either way, we need to use eslint-disable.